src/assets/translations/en/main.json

```diff
@@ -1319,5 +1319,13 @@
         "neverSellData": "Never sell data for profit"
       }
     }
+  },
```

Comment:
we only need to add the english translation - adding translations for the other languages actually makes it more difficult for @thesmithdao and @firebomb1 to know when things need updating.

can you please revert the changes to the other languages and just make these changes to the english files.